client/main.lua

```diff
@@ -2099,39 +2099,20 @@ end)
 ------------------------------refreshskin-------------------
 
 RegisterCommand("refreshskin", function(source, args, rawCommand)
-
-
-    local playerPed = PlayerPedId()
-    local maxhealth = GetEntityMaxHealth(playerPed)
-    local health = GetEntityHealth(playerPed)
-
-    reloadSkin(health)
-
+    	local ped = PlayerPedId()
+	local model = GetEntityModel(ped)
```

Comment:
Might just be me being a defensive programmer, but could you add some logic in the event that GetEntityModel(ped) returns null please?

Eg set model to mp_m_freemode_01/mp_f_freemode_01